src/event-bridge/event-bus.ts

```diff
@@ -120,9 +124,37 @@ abstract class EventBusBase<E extends EventBusRuleInput>
   readonly eventBusName: string;
   readonly eventBusArn: string;
 
+  readonly native: NativeIntegration<EventBusBase<E>>;
+
   constructor(readonly bus: aws_events.IEventBus) {
     this.eventBusName = bus.eventBusName;
     this.eventBusArn = bus.eventBusArn;
+
+    // Closure event bus base
+    const eventBusName = this.eventBusName;
+    this.native = <NativeIntegration<EventBusBase<E>>>{
+      bootstrap: (context: Function<any, any>) => {
```

Comment:
should this be called `configure` or `synth`? `bootstrap` may be a bit vague/ambiguous. Although, funnily enough, it's exactly what I called it in punchcard haha

Reply:
`bind`?

Reply:
Like it.